src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/VariantEvaluationProgramGroup.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+/**
+ * Tools that evaluate and refine variant calls, e.g. with annotations not offered by the engine".
```

Comment:
"annotations not offered by the engine" is super cryptic outside of the GATK context so will get pushback in Picard; let's rephrase to  e.g.:

Tools that evaluate and refine variant calls, e.g. by adding annotations that are not calculated during variant calling

(I'm not crazy about this formulation either but it's more explicit at least)

Reply:
Done.